packages/components/sort-select/stories/index.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+import { useArgs } from '@storybook/preview-api';
+
+/**
+ * Internal dependencies
+ */
+import SortSelect, { type SortSelectProps } from '..';
+
+export default {
+	title: 'Checkout Components/SortSelect',
+	component: SortSelect,
+	argTypes: {},
```

Comment:
Let's add some arg types here so that they appear in the description of the component here:

![Screenshot 2023-11-07 at 11 25 27](https://github.com/woocommerce/woocommerce-blocks/assets/3966773/4bd0813c-2ca1-4759-8615-1edf46b04ca3)

Mike has a good example [here](https://github.com/woocommerce/woocommerce-blocks/blob/add/sort-select-story/assets/js/base/components/cart-checkout/totals/coupon/index.tsx#L42)

Reply:
I also wonder why the defaults don't show, it should pick those up from the props I assumed?

Reply:
<img width="1036" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/b59a9d56-edc1-43ad-8439-ca368a76cbe3">

Thanks, I had them added, not sure where that commit went, maybe lost during branch switches. Re-added them.